charts/timed/values.yaml

```diff
@@ -231,6 +231,25 @@ postgresql:
     selector:
       app: postgresql
       role: slave
+  tls:
+    # postgresql.tls.enabled -- Enable TLS for Postgresql
+    enabled: false
+    # postgresql.tls.certificatesSecret -- Name of TLS secret
+    certificatesSecret: psql-server
+    # postgresql.tls.certFilename -- Cert file name
+    certFilename: "tls.crt"
+    # postgresql.tls.certKeyFilename -- Cert key filename
+    certKeyFilename: "tls.key"
+    certificate:
+      # postgresql.tls.certificate.name -- Name of certificate
+      name: psql-server
+      # postgresql.tls.certificate.dnsNames -- DNS names of certificate
+      dnsNames: []
+      # postgresql.tls.certificate.issuerRef -- Issuer ref
+      issuerRef:
+        group: cert-manager.io
+        kind: ClusterIssuer
+        name: letsencrypt-prod
```

Comment:
I'd like to see issuerRef configurable so we can also support other issuers.

Also, please add a default doc snippet so helm-docs doesn't dump json to the docs.